src/main/java/racingcar/global/util/Validator.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.global.util;
+
+import racingcar.global.enums.ErrorMessage;
+
+public class Validator {
+    private static Validator instance;
+
+    public static Validator getInstance() {
+        if (instance == null) {
+            instance = new Validator();
+        }
+        return instance;
+    }
+
+    public void validateName(String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException(
+                    ErrorMessage.INVALID_NAME_LENGTH.getMessage());
+        }
```

Comment:
매직넘버는 상수화하거나 분기 조건을 메서드로 분리해 주시면 의미를 더 명확하게 전달할 수 있을 것 같습니다 !

```suggestion
        if (name.length() > CAR_NAME_MAX_LENGTH) {
            throw new IllegalArgumentException(
                    ErrorMessage.INVALID_NAME_LENGTH.getMessage());
        }
```

```suggestion
        if (isOverMaxLength(name.length()) {
            throw new IllegalArgumentException(
                    ErrorMessage.INVALID_NAME_LENGTH.getMessage());
        }
```